### PR TITLE
Vote Page & links to it and Dynmap in header

### PR DESCRIPTION
**Pull request by BraxtBooth**
Created a simple vote page that we can add to or change as needed. You should add information about the "perks/rewards" of voting for the server, and possibly set background images as the possible rewards.

Dynmap link in the header just redirects to your dynmap. We could do an embedded Dynmap I'm sure but this way is super easy.

### CODE REVIEW

**Comment by BraxtBooth** [comment]
image
Pic of the vote page

File: vote.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE HTML>
+<!--
+	Landed by HTML5 UP
+	html5up.net | @ajlkn
+	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
+-->
+<html>
+
+<head>
+	<title>ComparatorCraftSMP | FAQ</title>
```
Comment (LagunaElectric) [inline vote.html:10]: Change "FAQ" to "Vote"

File: vote.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+				<header class="major">
+					<h2>Vote for ComparatorCraftSMP</h2>
+                    <h1>
+                        <!-- Put description of rewards of some sort -->
+                        Vote for our server for awesome in game cosmetics!
+                    </h1>
+				</header>
+
+				<!-- Content -->
+				<section id="content" }>
```
Comment (LagunaElectric) [inline vote.html:37]: You left a lone `}` in your section opener.

**Review by LagunaElectric — CHANGES_REQUESTED** [review]
Just a couple of changes needed.

**Review by LagunaElectric — APPROVED** [review]

**Pull request merged**